aspnetcore/fundamentals/target-aspnetcore.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Use ASP.NET Core APIs in a class library
+author: scottaddie
+description: Learn how to use ASP.NET Core APIs in a class library.
+ms.author: scaddie
+ms.custom: mvc
+ms.date: 12/03/2019
+no-loc: [Blazor]
+uid: fundamentals/target-aspnetcore
+---
+# Use ASP.NET Core APIs in a class library
```

Comment:
I feel like one thing that's missing here - a mention of our support lifecyle, and links to where library authors can find the current data about what's in support. 

Whenever I talk to community library authors I make it a point to bring this us. Most don't know that they are targeting release of ASP.NET Core that we've dropped support for - they are used to the old world where stuff lives forever.

Obviously we should boss people around, but making an effort to target all of the ASP.NET Core releases that are currently in support, and it's ok to ignore the ones that aren't seems like an OK suggestion.